examples/PhasorDynamics/Example1/example1.cpp

```diff
@@ -93,19 +130,19 @@ int main()
   // reference to that variable inside the callback). We select
   // the subset of the output we're interested in recording and
   // push it into output, which is updated outside the callback.
-  auto output_cb = [&](double t)
+  auto output_cb = [&](real_type t)
   {
-    std::vector<double>& yval = sys.y();
+    std::vector<scalar_type>& yval = sys.y();
```

Comment:
```suggestion
    std::vector<scalar_type>& y_val = sys.y();
```
And elsewhere perhaps?

Reply:
Addressed in [ab31957](https://github.com/ORNL/GridKit/pull/133/commits/ab319577ca89d52c745c86c73e842bd49c14241e).